ibm/service/project/data_source_ibm_project_event_notification_test.go

```diff
@@ -0,0 +1,102 @@
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+)
+
+func TestAccIbmProjectEventNotificationDataSourceBasic(t *testing.T) {
+	projectName := fmt.Sprintf("tf_name_%d", acctest.RandIntRange(10, 100))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheck(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIbmProjectEventNotificationDataSourceConfigBasic(projectName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "project_id"),
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "name"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccIbmProjectEventNotificationDataSourceAllArgs(t *testing.T) {
+	projectName := fmt.Sprintf("tf_name_%d", acctest.RandIntRange(10, 100))
+	projectDescription := fmt.Sprintf("tf_description_%d", acctest.RandIntRange(10, 100))
+	projectResourceGroup := fmt.Sprintf("Default")
+	projectLocation := fmt.Sprintf("us-south")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheck(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIbmProjectEventNotificationDataSourceConfig(projectName, projectDescription, projectResourceGroup, projectLocation),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "project_id"),
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "exclude_configs"),
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "complete"),
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "name"),
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "description"),
+					resource.TestCheckResourceAttrSet("data.ibm_project_event_notification.project_event_notification", "metadata.#"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIbmProjectEventNotificationDataSourceConfigBasic(projectName string) string {
+	return fmt.Sprintf(`
+		resource "ibm_project_instance" "project_instance_instance" {
+			name = "%s"
+		}
+
+		data "ibm_project_event_notification" "project_event_notification" {
+			project_id = ibm_project_instance.project_instance_instance.id
+			exclude_configs = false
+			complete = true
+		}
+	`, projectName)
+}
+
+func testAccCheckIbmProjectEventNotificationDataSourceConfig(projectName string, projectDescription string, projectResourceGroup string, projectLocation string) string {
+	return fmt.Sprintf(`
+		resource "ibm_project_instance" "project_instance_instance" {
+			name = "%s"
+			description = "%s"
+			configs {
+				id = "id"
```

Comment:
Can the value of `id` be just id?
Does API allow this? how did the testcase pass?

Reply:
done